charts/super-agent-deployment/templates/deployment-superagent.yaml

```diff
@@ -90,6 +95,11 @@ spec:
         {{- with .Values.extraVolumes }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
+        {{- if eq (include "newrelic-super-agent.shouldCreateAuthSecret" .) "true" }}
+        - name: auth-secret-volume
+          secret:
+            secretName: {{ .Values.authSecret.name | default .Release.Name }}
```

Comment:
Should the default has some sort of prefix? Would also make sense a default value in `values.yaml`?